app/(main)/league/[leagueId]/entry/[entryId]/week/Week.test.tsx

```diff
@@ -141,11 +174,11 @@ describe('Week', () => {
     );
   });
 
-  test('should show error notification when changing your team fails', async () => {
+  xtest('should show error notification when changing your team fails', async () => {
```

Comment:
are we omitting this test for now?

Reply:
Correct. These are testing the helper file directly, which should be reworked into a separate file.